packages/build-utils/src/workspaces/workspace-managers.ts

```diff
@@ -0,0 +1,65 @@
+import type { Framework } from '@vercel/frameworks';
+
+/**
+ * The supported list of workspace managers.
+ *
+ * This list is designed to work with the @see {@link detectFramework} function.
+ *
+ * @example
+ *   import { workspaceManagers as frameworkList } from '@vercel/build-utils/workspaces'
+ *   import { detectFramework } from '@vercel/build-utils'
+ *
+ *   const fs = new GitDetectorFilesystem(...)
+ *   detectFramwork({ fs, frameworkList }) // returns the 'slug' field if detected, otherwise null
+ *
+ * @todo Will be used by the detect-eligible-projects API endpoint for a given git url.
+ */
+export const workspaceManagers = [
+  {
+    name: 'yarn',
+    slug: 'yarn',
+    detectors: {
```

Comment:
I believe the order here matters so this should match what we do for the install command in the event when there are multiple managers detected.

<img width="584" alt="image" src="https://user-images.githubusercontent.com/229881/166571020-0fc2a013-0aea-43fc-8b3e-cc91f1c93824.png">

cc @Ethan-Arrowood who just worked on this

Reply:
agreed

Reply:
Good catch 🙌